mypy type errors after upgrading tenacity to 9.1.3: logging.Logger rejected

After upgrading Tenacity, mypy reports errors like:

```
error: Argument 1 to "before_sleep_log" has incompatible type "Logger"; expected "LoggerProtocol"  [arg-type]
```

I'm using the normal Python logging logger. 
This should already reproduce the error:
```python
  import logging
  from tenacity.before_sleep import before_sleep_log

  log = logging.getLogger(__name__)
  before_sleep_log(log, logging.INFO)  # mypy arg-type error
```

--- a/tenacity/_utils.py
+++ b/tenacity/_utils.py
@@ -33,7 +33,7 @@ class LoggerProtocol(typing.Protocol):
     """
 
     def log(
-        self, level: int, msg: str, /, *args: typing.Any, **kwargs: typing.Any
+        self, level: int, msg: str, *args: typing.Any, **kwargs: typing.Any
     ) -> typing.Any: ...
 
 

